### PR TITLE
Create root readme

### PR DESCRIPTION
**Pull request by KyDev25**
Create readme with contributors and Jira link.

### CODE REVIEW

**Review by BarnabePILLIAUDIN — APPROVED** [review]
LGMT

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+- Magali DESLOUS-PAOLI (Magali-D) magali.paoli@tutanota.com
+- Kylian BARTHELEMY (KyDev25) kylian.barthelemy2003@gmail.com
+- Cédric GAUTIER (cedricgautier) gautiercedric07@gmail.com
+- Barnabé PILLIAUDIN (BarnabePILLIAUDIN) barnabe.pilliaudin@gmail.com
+- Alix BERTHELOT (hallycsse) alixberthelot03@gmail.com
+
+# Jira link
+
+[Link Jira Here]https://kylianbarthelemy.atlassian.net/jira/software/projects/AIRNEIS/boards/2
```
Comment (cedricgautier) [inline README.md]:
```suggestion
[Tracker - Kanban board]https://kylianbarthelemy.atlassian.net/jira/software/projects/AIRNEIS/boards/2
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Contributors
+
+- Magali DESLOUS-PAOLI (Magali-D) magali.paoli@tutanota.com
+- Kylian BARTHELEMY (KyDev25) kylian.barthelemy2003@gmail.com
+- Cédric GAUTIER (cedricgautier) gautiercedric07@gmail.com
+- Barnabé PILLIAUDIN (BarnabePILLIAUDIN) barnabe.pilliaudin@gmail.com
+- Alix BERTHELOT (hallycsse) alixberthelot03@gmail.com
```
Comment (cedricgautier) [inline README.md]:
```suggestion
- Magali DESLOUS-PAOLI (Magali-D) <magali.paoli@tutanota.com>
- Kylian BARTHELEMY (KyDev25) <kylian.barthelemy2003@gmail.com>
- Cédric GAUTIER (cedricgautier) <gautiercedric07@gmail.com>
- Barnabé PILLIAUDIN (BarnabePILLIAUDIN) <barnabe.pilliaudin@gmail.com>
- Alix BERTHELOT (hallycsse) <alixberthelot03@gmail.com>
```

**Review by cedricgautier — COMMENTED** [review]
Format & correct

**Review by cedricgautier — APPROVED** [review]

**Review by Magali-D — APPROVED** [review]

**Pull request merged**